src/pages/home/report/ReportActionsView.js

```diff
@@ -453,13 +421,4 @@ export default compose(
     withDrawerState,
     withLocalize,
     withNetwork(),
-    withOnyx({
```

Comment:
How does the component get `this.props.report` now without using `withOnyx()`?

Reply:
from the parent component here:

https://github.com/Expensify/App/blob/972b83549e10cf1f05310f269f07a2a83b1e2253/src/pages/home/ReportScreen.js#L242-L245

Reply:
Oh, Thanks! I don't think we should pass both the `reportID` and `report`. Those are redundant. Let's just pass `report` and get the `reportID` off that.

Reply:
I can do that, but reportID at the moment is `required` not sure if we should make the `report` required as well,  not sure if we should have a default value for reportID

Reply:
This sounds like an easy change and a good thing to look into. I also think it would be OK to do in a follow up and not add additional refactoring scope to these changes. I don't immediately see a clear reason for passing the `reportID` and `report` as separate props, but I haven't looked into it deeply enough to say there isn't one.

Reply:
I think we should make `report` the required prop then, but as always, make sure you take a look at the code and understand it first before making a change like that.

I won't fight tooth-and-nail about it being done in this PR, but I also think this is a refactor PR and I am not too concerned about scope for something small like this.

Reply:
just made the change and tested it after, seems to work great, let me know if you are happy with that @tgolen